notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Modified 'Vertex SDK' to 'Vertex AI SDK'